engine/src/main/java/org/hibernate/search/util/logging/impl/Log.java

```diff
@@ -736,4 +737,6 @@
 	@Message(id = 248, value = "WorkList should never be empty. Stacktrace below \n %s" )
 	void workListShouldNeverBeEmpty(String stackTrace);
 
+	@Message(id = 249, value = "Unsupported value type for configuration property " + Environment.ERROR_HANDLER + ": %1$s")
+	SearchException unsupportedErrorHandlerConfigurationValueType(@FormatWith(ClassObjectFormatter.class) Class<?> errorHandlerValueType);
```

Comment:
:+1: 

I didn't mean to make it a big discussion.. just curious.
I would object though about introducing pointless code like a `toString()` service :)
But it doesn't seem the case, I like your idea now that I'm not startled by the surprise of not having a clue about that annotation.